lib/src/base-styles/_nx-colors.scss

```diff
@@ -368,6 +380,15 @@
     --nx-color-checkbox-disabled-background: var(--nx-color-disabled);
     --nx-color-checkbox-disabled-inner-border: var(--nx-color-disabled);
     --nx-color-checkbox-disabled-icon: var(--nx-color-component-background);
+
+    // NxTransferList
+    --nx-color-transfer-list-item-background: var(--nx-swatch-indigo-20);
+    --nx-color-transfer-list-item-background-hover: var(--nx-swatch-indigo-15);
+    --nx-color-transfer-list-item-background-active: var(--nx-swatch-indigo-35);
```

Comment:
In the sketch, the color for background active is set to `nx-indigo-30`

Reply:
> In the sketch, the color for background active is set to nx-indigo-30

465f00d2e